From f8d75a73b407f1859752fee09ef8d86fc2a85193 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Thu, 7 Apr 2022 04:58:39 +0200
Subject: [PATCH 45/49] arm64: dts: imx8mp: Enable weak pullup until R269 is
 populated

External RTC IRQ line pull up R269 is not populated on the prototype
boards, enable SoC pull up instead to avoid having the IRQ line in
undefined state.

Upstream-Status: Inappropriate [Drop once HW is fixed]
Signed-off-by: Marek Vasut <marex@denx.de>
Cc: Fabio Estevam <festevam@denx.de>
Cc: Peng Fan <peng.fan@nxp.com>
Cc: Shawn Guo <shawnguo@kernel.org>
Cc: NXP Linux Team <linux-imx@nxp.com>
To: linux-arm-kernel@lists.infradead.org
---
 arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi
index cd23b903180e3..d1d3b7c530fb7 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-dhcom-som.dtsi
@@ -835,7 +835,7 @@ MX8MP_IOMUXC_GPIO1_IO01__PWM1_OUT		0x6
 	pinctrl_rtc: dhcom-rtc-grp {
 		fsl,pins = <
 			/* RTC_#INT Interrupt */
-			MX8MP_IOMUXC_SPDIF_EXT_CLK__GPIO5_IO05		0x40000080
+			MX8MP_IOMUXC_SPDIF_EXT_CLK__GPIO5_IO05		0x400001c6
 		>;
 	};
 
-- 
2.35.1

